test/test_intensify/test_intensify.py

```diff
@@ -804,7 +804,8 @@ def target(x):
         # the basis configuration (config3) not better than incumbent, so can move on
         self.assertEqual(inc, self.config1)
         self.assertEqual(self.stats.inc_changed, 2)
-        self.assertEqual(intensifier.stage, IntensifierStage.RUN_INCUMBENT)
+        self.assertEqual(intensifier.stage, IntensifierStage.RUN_INCUMBENT,
+                         msg=self.rh.data.items())
```

Comment:
I assume this is only for debugging?

Reply:
Yes! because this check was failing and I could not understand why. So I let this for the future in case this happens again. This was solved via --forked.